pkg/migration/migrations/separated_intents.go

```diff
@@ -90,17 +98,21 @@ type migrateLockTablePool struct {
 	done     chan bool
 	status   [concurrentMigrateLockTableRequests]int64
 	finished uint64
```

Comment:
Oversight on my part, but we could probably benefit with a comment on `finished` saying it should be updated atomically.